libs/sdk-ui-ext/src/internal/interfaces/Visualization.ts

```diff
@@ -405,6 +408,12 @@ export interface IVisualization {
      * @returns `source` as the Drill Down target {@link IInsight}
      */
     getInsightWithDrillDownApplied(source: IInsight, drillDownContext: IDrillDownContext): IInsight;
+
+    /**
+     * Called whenever inputs for default sorts calculation changed and it should provide current sort and also all valid available sorts for actual reference point/properties
+     * @param referencePoint - the reference point for which the sort config needs to be evaluated
+     */
+    getSortConfig(referencePoint: IReferencePoint): Promise<ISortConfig>;
```

Comment:
Should the sortConfig be part of the extended reference point and this handled in getExtendedReferencePoint? Or why is this a separate function?

Reply:
@lupko wished that. It allows us to have these life cycles separated and smallet. And also there is a risk, that whole sortConfig evaluation will need to wait for execution result in the future, when we will incorporate attr. elements into possible sorts. But it is out of scope for now

Reply:
Thats because Visualisation could report possible sorts after first ex. Same as drills.

Reply:
Oh ok, makes sense :) thanks